Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting.yml

```diff
@@ -0,0 +1,1496 @@
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.time
+      description: The time in 24 hour format based on the timezone parameter.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.date
+      description: The date from the timestamp based on the timezone parameter.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.identities.id
+      description: ID of identity.
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.identities.type.id
+      description: Origin type for identity
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.identities.type.type
+      description: Origin type name for identity
+      type: String
+    - contextPath: UmbrellaReporting.Activity.identities.type.label
+      description: Origin type label for identity
+      type: String
+    - contextPath: UmbrellaReporting.Activity.identities.label
+      description: Label for identity
+      type: String
+    - contextPath: UmbrellaReporting.Activity.identities.deleted
+      description: Indicates whether the identity was deleted or not
+      type: Boolean
+    - contextPath: UmbrellaReporting.Activity.threats.label
+      description: The threat name or label.
+      type: Boolean
+    - contextPath: UmbrellaReporting.Activity.threats.type
+      description: The type of threat.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allapplications.id
+      description: ID of the application.
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.allapplications.type
+      description: Type of the application, NBAR or AVC.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allapplications.label
+      description: Label of the application.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allapplications.category.label
+      description: Label of the application category.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allapplications.category.id
+      description: ID of the application category.
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.allowedapplications.id
+      description: ID of the application.
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.allowedapplications.label
+      description: Label of the application.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allowedapplications.type
+      description: Type of the application, NBAR or AVC.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allowedapplications.category.label
+      description: Label of the application category.
+      type: String
+    - contextPath: UmbrellaReporting.Activity.allowedapplications.category.id
+      description: ID of the application category.
+      type: Number
+    - contextPath: UmbrellaReporting.Activity.querytype
+      description: The type of DNS request that was made. For more information, see Common DNS Request Types.
```

Comment:
Need a link here to Common DNS Request Types.

Reply:
@RamboV - any update about that comment?

Reply:
@ShacharKidor Link Added